projects/map-ol/src/lib/map-ol.service.ts

```diff
@@ -1602,6 +1637,7 @@ export class MapOlService {
    */
   private destroyDynamicPopupComponent(id: string): void {
     if (this.dynamicPopupComponents.has(id)) {
+      const comp = this.dynamicPopupComponents.get(id);
```

Comment:
what is this variable `const comp` used for? 
I only see it declared but never used then.

Reply:
Good catch - I've removed that declaration.